Packs/MongoDB/Integrations/MongoDB/README.md

```diff
@@ -326,6 +327,76 @@ There is no context output for this command.
 ##### Human Readable Output
 ### MongoDB: Collection 'collectionToDelete` has been dropped.
 
+### mongodb-pipeline-query
+***
+Searches for items by the specified JSON pipleline query.
+
+
+#### Base Command
+
+`mongodb-pipeline-query`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| collection | Name of the collection to query. | Required | 
+| pipeline | A JSON pipeline query to search by for in the collection. Pipeline query should by list of dictionaries. For example: [{"key1": "value1"}, {"key2": "value2"}]. | Required | 
+| limit | Limits the result returned from MongoDB. Default value - 50. Default is 50. | Optional | 
+| offset | Offset of the result returned from MongoDB. Default value - 0. Default is 0. | Optional | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| MongoDB.Entry._id | String | The ID of entry from the query. | 
+| MongoDB.Entry.collection | String | The collection of which the entry belongs to. | 
+
+
+#### Command Example
+```!mongodb-pipeline-query collection=test_collection pipeline="[{\"$match\": {\"title\": \"test_title\"}}]"```
+
+#### Context Example
+```json
+{
+    "MongoDB": {
+        "Entry": [
+            {
+                "_id": "602e624e8be6cb93eb795695",
```

Comment:
I think you should remove this IDs

Reply:
previous commands at the readme has ID's too.